cg/meta/workflow/fluffy.py

```diff
@@ -20,6 +28,51 @@
 LOG = logging.getLogger(__name__)
 
 
+class FluffySampleSheetHeaders(Enum):
```

Comment:
I prefer keeping them as is since they are only used here
```python
headers = [header.value for header in FluffySampleSheetHeaders]
```
And all caps variables are stressful haha 😅

Reply:
Apparently we will most likely remove this config completely https://github.com/Clinical-Genomics/cg/pull/2458#issuecomment-1715524346 so I don't think the constants are likely to be used elsewhere